src/mtx3/utils/CommandBinaryBuffer.ts

```diff
@@ -7,14 +7,1284 @@
+    RELAY_ON_TARIFF_3: boolean;
+
+    /**
+     * Turn on by restoration of good voltage.
+     */
+    RELAY_ON_V_GOOD: boolean;
+
+    /**
+     * Relay deactivation function (`1` - enabled, `0` - disabled).
+     */
+    RELAY_OFF_Y: boolean;
+
+    /**
+     * Turn off by command from the center.
+     */
+    RELAY_OFF_CENTER: boolean;
+
+    /**
+     * Turn off by tariff `T1`.
+     */
+    RELAY_OFF_TARIFF_0: boolean;
+
+    /**
+     * Turn off by tariff `T2`.
+     */
+    RELAY_OFF_TARIFF_1: boolean;
+
+    /**
+     * Turn off by tariff `T3`.
+     */
+    RELAY_OFF_TARIFF_2: boolean;
+
+    /**
+     * Turn off by tariff `T4`.
+     */
+    RELAY_OFF_TARIFF_3: boolean;
+
+    /**
+     * Turn off on load current exceeding.
+     */
+    RELAY_OFF_I_LIMIT: boolean;
+
+    /**
+     * Turn off on poor voltage.
+     */
+    RELAY_OFF_V_BAD: boolean;
+
+    /**
+     * Turn off on exceeding the differential current limit.
+     */
+    RELAY_OFF_DIFF_BAD: boolean;
+
+    /**
+     * Turn off on exceeding the power consumption limit for tariff `T1`.
+     */
+    RELAY_OFF_LIM_TARIFF_0: boolean;
+
+    /**
+     * Turn off on exceeding the power consumption limit for tariff `T2`.
+     */
+    RELAY_OFF_LIM_TARIFF_1: boolean;
+
+    /**
+     * Turn off on exceeding the power consumption limit for tariff `T3`.
+     */
+    RELAY_OFF_LIM_TARIFF_2: boolean;
+
+    /**
+     * Turn off on exceeding the power consumption limit for tariff `T4`.
+     */
+    RELAY_OFF_LIM_TARIFF_3: boolean;
+
+    /**
+     * Turn off on exceeding the reactive power consumption limit for tariff `T1`.
+     */
+    RELAY_OFF_LIM_VAR_TARIFF_0: boolean;
+
+    /**
+     * Turn off on exceeding the reactive power consumption limit for tariff `T2`.
+     */
+    RELAY_OFF_LIM_VAR_TARIFF_1: boolean;
+
+    /**
+     * Turn off on exceeding the reactive power consumption limit for tariff `T3`.
+     */
+    RELAY_OFF_LIM_VAR_TARIFF_2: boolean;
+
+    /**
+     * Turn off on exceeding the reactive power consumption limit for tariff `T4`.
+     */
+    RELAY_OFF_LIM_VAR_TARIFF_3: boolean;
+
+    /**
+     * Turn on when `cos φ` is normalized.
+     */
+    RELAY_ON_PF_MIN: boolean;
+
+    /**
+     * Turn off on `cos φ`.
+     */
+    RELAY_OFF_PF_MIN: boolean;
+
+    /**
+     * Turn on after timeout.
+     */
+    RELAY_ON_TIMEOUT: boolean;
+
+    /**
+     * Turn on based on saldo.
+     */
+    RELAY_ON_SALDO: boolean;
+
+    /**
+     * Turn off based on saldo.
+     */
+    RELAY_OFF_SALDO: boolean;
+
+    /**
+     * Turn off based on saldo with condition.
+     */
+    RELAY_OFF_SALDO_SOFT: boolean;
+}
+
+export interface ISpeedOptoPortOperatorParameter {
+    /**
+     * Baud rate of the PLC (UART_0): `2400` or `9600`.
+     *
+     * | Value | Baud Rate |
+     * |-------|-----------|
+     * | 0     | 9600      |
+     * | 2     | 2400      |
+     * | 4     | 9600      |
+     */
+    plc: typeof baudRates.RATE_2400 | typeof baudRates.RATE_9600,
```

Comment:
is it useful?

seems like `plc: number | number`

Reply:
![image](https://github.com/jooby-dev/jooby-codec/assets/44006743/d34ef513-585b-4b04-b952-47996cb5db63)